src/xPDO/xPDO.php

```diff
@@ -147,9 +148,9 @@ class xPDO {
      */
     private $cachePath= null;
     /**
-     * @var array An array of supplemental service classes for this xPDO instance.
+     * @var xPDOContainer A container of supplemental service classes for this xPDO instance.
      */
-    public $services= array ();
+    public $services= null;
```

Comment:
Isn't changing the type some kind of breaking change (it's a public attribute) ?
Albeit breaking things is supposed to be "allowed" (major version bump), shouldn't this be a new attribute ?